mopidy/install.sh

```diff
@@ -193,11 +188,40 @@ echo
 # MAYBE?: Remove the sources.list to avoid any future issues with apt.mopidy.com failing
 # rm -f /etc/apt/sources.list.d/mopidy.list
 
```

Comment:
I see some things in the service's logs. Mainly:
* `Ignoring config section 'spotify' because no matching extension was found`
  * Suggestion: remove the `spotify` section in `mopidy.conf` and the messaging about Spotify account details at the end of the script
* `Ignoring config section 'mpd' because no matching extension was found`
  * Suggestion: remove the `mpd` section from `mopidy.conf`
* `/home/pi/Music is not a directory. Please create the directory or update the file/media_dirs config value.`
  * I think we should `mkdirp /home/$MOPIDY_USER/Music` somewhere in the install script